libs/angular-accelerator/src/lib/components/interactive-data-view/interactive-data-view.component.ts

```diff
@@ -315,21 +338,67 @@ export class InteractiveDataViewComponent implements OnInit, AfterContentInit {
     this._data = value
   }
 
+  columnGroupSlotName = 'onecx-shell-column-group-selection'
+  isColumnGroupSelectionComponentDefined$: Observable<boolean>
+  groupSelectionChangedSlotEmitter = new EventEmitter<ColumnGroupData | undefined>()
+
+  constructor(private slotService: SlotService) {
+    this.isColumnGroupSelectionComponentDefined$ = this.slotService
+      .isSomeComponentDefinedForSlot(this.columnGroupSlotName)
+      .pipe(startWith(true))
+
+    this.groupSelectionChangedSlotEmitter.subscribe((event: ColumnGroupData | undefined) => {
+      if (event === undefined) {
+        event = {
+          activeColumns: this.displayedColumns,
+          groupKey: this.selectedGroupKey ?? this.defaultGroupKey,
+        }
+      }
+      this.displayedColumnKeys$.next(event.activeColumns.map((col) => col.id))
+      this.selectedGroupKey$.next(event.groupKey)
+      // TODO: Remove following line once displayedColumns (deprecated) has been removed
```

Comment:
Remove the comment. When displayedColumns + displayedColumnsChange are removed, the compiler will tell you to remove this